examples/heterogeneous-hello-world/shader.slang

```diff
@@ -3,9 +3,8 @@
 //TEST_INPUT:ubuffer(random(float, 4096, -1.0, 1.0), stride=4):name=ioBuffer
 RWStructuredBuffer<float> ioBuffer;
 
-[shader("compute")]
 [numthreads(4, 1, 1)]
-void computeMain(uint3 dispatchThreadID : SV_DispatchThreadID)
+public void computeMain(uint3 dispatchThreadID : SV_DispatchThreadID)
```

Comment:
I'm not sure that changing this signature in the way that was done here is exactly what we want, but I suppose it doesn't really matter to the implementation of the feature.